brainstormforce/ultimate-addons-for-gutenberg
# Team - Acceptance Testing Round 2 Changes.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/2806a80d-012d-41a6-ac56-09d16b6311eb?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- Screencast added in task comment.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Team block.
- Check for String changes.
- Image Vertical Alignment (Top/Middle when the image width is full or max)
- Check block on the responsive device when an image is added.
- Default value changes (Check Backword case here)
- Check for console and debug log.

### Checklist:
